python/strings_udf/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do we need this file here? We don't include separate license files for `cudf`, `dask_cudf`, etc. We just have one license in the repo root.

Reply:
Removed